### ISSUE TITLE
average coherence

### ISSUE THREAD

**Issue by faraksoch1881** [yumorishita/LiCSBAS#409]
Hi Dr. Yumorishit, 

I processed the files and obtained the coh_avg.png inside the results folder and it has a scale bar that goes upto 0.9. But inthe text file 11fg_stats.txt, the column coh_av contains the value less then 0.4. So could you please explain it or is there any other files i can find the information of average coherence related to the image i obtained. 

Thank you 
Also, 
![11fg_stats](https://github.com/user-attachments/assets/e5fe0c3e-b7bc-42fd-8499-3749ba80988d)
![coh_avg](https://github.com/user-attachments/assets/bcd9b504-0d20-46ef-82c6-6e52928eeddc)

**Comment by yumorishita** [comment]
coh_avg.png shows the average coherence for each pixel from all ifgs, but coh_av in 11fg_stats.txt is the average for each ifg from all pixels. They are different.

**Comment by faraksoch1881** [comment]
Is there a file where coh_avg value is saved from which coh_avg.png was created ?

**Comment by MuhammadAsi** [comment]
Dear Dr. Yumorishita,

I am currently working in the Himalayan region and encountering challenges such as radar shadows, layovers, and foreshortening due to the high elevation. I would like to ask for your advice on the appropriate coherence threshold to use to avoid misinterpreting these no-data areas as deformation.

I understand that a coherence threshold of 0.7 is generally recommended, but as I am new to LiCSBAS, I am uncertain which specific threshold to apply in this context. Should I set the coherence threshold to 0.7?

Additionally, in the 11ifg_stats.txt file, I have noticed that the average coherence (Coh_Av) is mostly below 0.4. I would appreciate your guidance on how to proceed given this situation.

Thank you for your assistance.

Thank you.
p04_mask_coh_thre=""	# e.g. 0.2
p11_coh_thre=""	# default: 0.05
p15_coh_thre=""	# default: 0.05
